src/NServiceBus.Core/Routing/FileBasedDynamicRouting/FileRoutingTableFeature.cs

```diff
@@ -11,19 +12,31 @@ public FileRoutingTableFeature()
             {
                 s.SetDefault(CheckIntervalSettingsKey, TimeSpan.FromSeconds(30));
                 s.SetDefault(MaxLoadAttemptsSettingsKey, 10);
+                s.SetDefault(FilePathSettingsKey, "instance-mapping.xml");
             });
             DependsOn<RoutingFeature>();
         }
 
         protected internal override void Setup(FeatureConfigurationContext context)
         {
             var filePath = context.Settings.Get<string>(FilePathSettingsKey);
+            if (!Path.IsPathRooted(filePath))
+            {
+                filePath = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, filePath);
+            }
+
+            if (!File.Exists(filePath))
+            {
+                return;
+            }
+
             var checkInterval = context.Settings.Get<TimeSpan>(CheckIntervalSettingsKey);
             var maxLoadAttempts = context.Settings.Get<int>(MaxLoadAttemptsSettingsKey);
 
             var endpointInstances = context.Settings.Get<EndpointInstances>();
 
             var fileRoutingTable = new FileRoutingTable(filePath, checkInterval, new AsyncTimer(), new RoutingFileAccess(), maxLoadAttempts);
+            fileRoutingTable.ReloadData();
```

Comment:
By calling this here, you're trying to reintroduce a setup check instead on first message send, right?

Currently, this means the underlying exception is going to bubble up instead of being wrapped in with the  `errorMessage` that used to be used everywhere.